lib/questionnaires/model/src/questionnaire_item_model.dart

```diff
@@ -20,6 +21,7 @@ class QuestionnaireItemModel with Diagnosticable {
   final QuestionnaireItemModel? parent;
   final QuestionnaireModel questionnaireModel;
   final int level;
+  final Locale? locale;
```

Comment:
> Maybe we can get the locale directly from this.questionnaireModel.locale

Thanks @fjwong!

TBH, I don't really remember why I pass `locale` to `QuestionnaireItemModel`.

In any case, after checking again the implementation, I agree it would be better to remove `locale` from `QuestionnaireItemModel`.